src/main/scala/viper/gobra/reporting/Message.scala

```diff
@@ -46,7 +46,12 @@ case class GobraOverallFailureMessage(verifier: String, result: VerifierResult)
     s"failure=${result.toString})"
 }
 
-case class GobraEntitySuccessMessage(verifier: String, concerning: Source.Verifier.Info) extends GobraVerificationResultMessage {
+sealed trait GobraEntityResultMessage extends GobraVerificationResultMessage {
```

Comment:
What is the purpose of storing `entity` here? I might have missed something but it does not seem to be used by any of its subcalsses

Reply:
It allows Gobra Server to figure out to which entity a message belongs. In particular, it keeps track of the progress by having a list of entities extracted from the Viper AST and checking whether an entity message belongs to any of these

Reply:
I see, you use it to keep track of the progress in the GobraIDE